MiroWindowsManager.spoon/init.lua

```diff
@@ -201,6 +207,18 @@ function obj:resize(growth)
 
   fr = fr:intersect(frontmostScreen():frame())  -- avoid sizing out of bounds
 
+  if self.stickySides then
```

Comment:
ooooo, nice! I would love to see this get merged so I can get back on the main branch. cc: @miromannino